deepchem/utils/sequence_utils.py

```diff
@@ -1,6 +1,7 @@
 from logging import raiseExceptions
 import os
 import subprocess
+from bio import SeqIO
```

Comment:
This should be in a try except block or SeqIO should be imported locally in a function where it is used